src/main/java/net/openhft/chronicle/wire/WireMarshaller.java

```diff
@@ -755,9 +751,10 @@ protected void getValue(Object o, @NotNull ValueOut write, Object previous) {
         @Override
         protected void setValue(Object o, @NotNull ValueIn read, boolean overwrite) {
             @NotNull StringBuilder sb = unsafeGetObject(o, offset);
-            if (sb == null)
+            if (sb == null) {
                 sb = new StringBuilder();
                 unsafePutObject(o, offset, sb);
+            }
```

Comment:
I noticed this when I auto-formatted. The indenting seemed to match the intention, but the lack of brackets meant we always `unsafePut..` the string buffer, even if `unsafeGet..` returned non-null.

Reply:
We should always use brackets in Java.